### PR TITLE
update the TOS accept body description and add fields to the Enabled entity

### PR DESCRIPTION
**Pull request by jmthibault79**
A small update to match the changes made when I adapted it to AoU here: https://github.com/all-of-us/workbench/pull/6454

\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -8138,6 +8138,12 @@ components:
         allUsersGroup:
           type: boolean
           description: User is a member of the "All Users" group?
+        adminEnabled:
+          type: boolean
+          description: User is a member of the admin group?
```
Comment (MatthewBemis) [inline src/main/resources/swagger/api-docs.yaml:8143]: I was confused by the naming of this `adminEnabled` field when I first saw it, but it's actually to indicate whether or not a Terra admin has explicitly disabled a user in the system. So maybe a more fitting description would be "User is enabled in the system?" or something similar

**Review by MatthewBemis — APPROVED** [review]
lgtm pending re-wording the `adminEnabled` description

File: src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (jmthibault79) [reply src/main/resources/swagger/api-docs.yaml:8143]: thanks, I was just guessing.  Updated.

Comment (jmthibault79) [reply src/main/resources/swagger/api-docs.yaml:8143]: ... but somehow it got lost in the squash/merge.  Trying again: #926

**Pull request merged**